packages/gatsby-theme/src/components/Contact/index.js

```diff
@@ -0,0 +1,8 @@
+import Contact from "./Contact"
+import ContactGrid from "./ContactGrid"
+import ContactHeader from "./ContactHeader"
+
+Contact.Grid = ContactGrid
+Contact.Header = ContactHeader
+
+export default Contact
```

Comment:
Please add this component to the [component index file](https://github.com/octahedroid/drupal-gatsby-theme/blob/master/packages/gatsby-theme/src/components/index.js)

Reply:
done @luisduenas